src/debug.c

```diff
@@ -1087,6 +1089,18 @@ NULL
             addReplySubcommandSyntaxError(c);
             return;
         }
+    } else if (!strcasecmp(c->argv[1]->ptr,"force-full-sync") && c->argc == 2) {
+        int ret = 0;
+        if (server.masterhost) {
+            ret = replicationForceFullSyncReplica();
```

Comment:
to be honest, at first glance, i mixed up `replicationForceFullSyncReplica` and `replicationForceFullSyncMaster`. i have thought `replicationForceFullSyncReplica` is for master
how about,
For master,`forceFullResyncToReplica()`,  `forceFullResyncFromMaster()` for replica